x-pack/platform/plugins/shared/lens/public/react_embeddable/initializers/initialize_edit.tsx

```diff
@@ -83,16 +84,16 @@ export function initializeEditApi(
     extractInheritedViewModeObservable(parentApi)
   );
 
-  const { disabledActionIds, setDisabledActionIds } = apiPublishesDisabledActionIds(parentApi)
+  const { disabledActionIds$, setDisabledActionIds } = apiPublishesDisabledActionIds(parentApi)
     ? parentApi
-    : { disabledActionIds: undefined, setDisabledActionIds: noop };
-  const [disabledActionIds$, disabledActionIdsComparator] = buildObservableVariable<
-    string[] | undefined
-  >(disabledActionIds);
+    : {
+        disabledActionIds$: new BehaviorSubject<string[] | undefined>(undefined),
```

Comment:
nit: I would prefer the previous version with `buildObservableVariables` that builds also the comparator.